docs/api/api.md

```diff
@@ -0,0 +1,168 @@
+# Saga API
+### Post transaction and compensation requests to Saga
+```
+POST /requests
+```
+
+####  Description
+
+1. Define requests in order and recovery policy by JSON format as below,put them to body.
+```
+{
+  "policy": "",
+  "requests": [
+    {
+      "id": "",
+      "type": "",
+      "serviceName": "",
+      "parents": [
+
+      ],
+      "transaction": {
+        "method": "",
+        "path": "",
+        "params": {
+
+        }
+      },
+      "compensation": {
+        "method": "",
+        "path": "",
+        "params": {
+
+        }
+      }
+    }
+  ]
+}
+```
+JSON parameters:
+- policy - support `BackwardRecovery` or `ForwardRecovery`.
+- requests - transactions array.
+  - id - user-defined service id.

```

Comment:
request id instead of service id. it shall be unique among this collection of requests

Reply:
modified completed